healvis/beam_model.py

```diff
@@ -316,6 +322,8 @@ def beam_val(self, az, za, freqs, **kwargs):
         """
         if self.beam_type == 'uniform':
             if isinstance(az, np.ndarray):
+                if np.isscalar(freqs):
```

Comment:
I don't think this is doing what you expect b/c its nested within an effective `is np.ndarray`

Reply:
This came up when I was running a single-frequency simulation with a uniform beam. There's no check that `freqs` is an ndarray, even if az/za are.

Reply:
oh sorry, i misread the line above as `freqs` too.